service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java

```diff
@@ -241,10 +235,6 @@ private String formatDateTimeType(DateTimeType dateTimeType) {
     }
 
     private String formatShortDate(Date date) {
-        return DATE_TIME_FORMATTER_SHORT
-            .format(date
-                .toInstant()
-                .atZone(ZoneId.of(UK_ZONE_ID))
-                .toLocalDateTime());
+        return new SimpleDateFormat("yyyy-MM-dd").format(date);
```

Comment:
We need to keep UK time zone, i'm fixing the issue in 1060

Reply:
This is also the wrong date format for the GP2GP HL7